queries/goctl/indents.scm

```diff
@@ -7,27 +7,10 @@
   (serviceSpec)
 ] @indent.begin
 
-(importGroup
-  ")" @indent.branch)
-
-(structType
-  "}" @indent.branch)
-
-(typeGroupSpec
-  ")" @indent.branch)
-
-(infoStatement
-  ")" @indent.branch)
-
-(serviceServerSpec
-  ")" @indent.branch)
-
-(serviceSpec
-  "}" @indent.branch)
-
 [
-  "}"
   ")"
-] @indent.end
+  "}"
+  "]"
```

Comment:
Not sure about this `]`. The 6 nodes declared above doesn't have this as a bracket.